pom.xml

```diff
@@ -53,7 +54,7 @@
         <hbase.version>1.4.0</hbase.version>
         <hadoop-two.version>2.7.5</hadoop-two.version>
         <curator.version>2.12.0</curator.version>
-        <phoenix.version>4.15.0-HBase-1.4-SNAPSHOT</phoenix.version>
+        <phoenix.version>4.15.0-HBase-1.3-SNAPSHOT</phoenix.version>
```

Comment:
This should stay to `4.15.0-HBase-1.4-SNAPSHOT` version. Please don't update it.